### PR TITLE
New Fenix telemetry for collapsing/expanding Inactive tabs

### PR DESCRIPTION
**Pull request by Mugurell**
Glean annotations for the telemetry added in https://github.com/mozilla-mobile/fenix/pull/20508

### CODE REVIEW

File: annotations/fenix/metrics/tabs_tray.inactive_tabs_collapsed/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+tags:
+  - Tabs
+---
+
+A user tapped the "Inactive tabs" header to collapse the group of inactive tabs.
```
Comment (wlach) [inline annotations/fenix/metrics/tabs_tray.inactive_tabs_collapsed/README.md:6]: This is just a duplication of the metric description, better to just leave it blank if you don't have anything to add here. 

Just to clarify, here's how the Glean Dictionary currently renders things (the description on top comes directly from the metrics.yaml files defined in Fenix):

https://dictionary.telemetry.mozilla.org/apps/fenix/metrics/tabs_tray_closed?channel=org.mozilla.firefox&ping=events

```suggestion
```

File: annotations/fenix/metrics/tabs_tray.inactive_tabs_expanded/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+tags:
+  - Tabs
+---
+
+A user tapped the "Inactive tabs" header to expand the group of inactive tabs.
```
Comment (wlach) [inline annotations/fenix/metrics/tabs_tray.inactive_tabs_expanded/README.md:6]: Same as above.

```suggestion
```

**Review by wlach — CHANGES_REQUESTED** [review]
Looks good except for the redundant descriptions. Can you apply the suggested changes and ask for review again?

File: annotations/fenix/metrics/tabs_tray.inactive_tabs_collapsed/README.md (hunk 1 above)
Comment (Mugurell) [reply annotations/fenix/metrics/tabs_tray.inactive_tabs_collapsed/README.md]: Thank you! Will keep this in mind for future work.

**Review by wlach — APPROVED** [review]

**Pull request merged**